app/renderer/abr-document.js

```diff
@@ -412,7 +478,7 @@ AbrDocument.prototype = {
     autopreview: function (types, lines) {
         var cm = this.cm;
         types = types || this.autopreviewTypes;
-        if (lines == null) {
+        if (lines === null) {
```

Comment:
Please don't change `== null`. This is why .jshintrc includes the ` "eqnull": true` rule.
See http://marijnhaverbeke.nl/blog/null-and-undefined.html for an explanation.